test/core/utils/preload.js

```diff
@@ -0,0 +1,45 @@
+
+
+describe('axe.utils.preload', function () {
```

Comment:
You're only testing that a queue is returned, not that it actually loaded anything.